lib/fluent/msgpack_factory.rb

```diff
@@ -70,9 +70,15 @@ def self.unpacker(*args)
       factory.unpacker(*args)
     end
 
-    def self.init
+    def self.init(enable_time_support: false)
```

Comment:
Should we register `MessagePack::Timestamp::TYPE` in `self.factory` as well? Or it needn't to be registered there? :thinking:

Reply:
Ah, I forgot `self.factory` because it is used in only tests...